todo_app/data/session_items.py

```diff
@@ -27,8 +26,8 @@ def get_item(id):
         item: The saved item, or None if no items match the specified ID.
     """
     items = get_items()
-    return next((item for item in items if item['id'] == int(id)), None)
 
+    return next((item for item in items if item['id'] == id), None)
```

Comment:
App is currently broken since you introduced the Task class:

```suggestion
    return next((item for item in items if item.id == id), None)
```

Similarly in other places in this file.